tests/Feature/Api/UsersTest.php

```diff
@@ -196,14 +196,11 @@ public function testGetUser()
    */
   public function testUpdateUserParametersRequired()
   {
-      $faker = Faker::create();
-
-      $uuid = factory(User::class)->create(['username' => 'mytestusername'])->uuid_text;
       //The post must have the required parameters
-      $url = self::API_TEST_URL . '/' .$uuid;
+      $url = self::API_TEST_URL . '/' . factory(User::class)->create()->uuid_text;
 
       $response = $this->apiCall('PUT', $url, [
-          'username' => 'updatemytestusername'
```

Comment:
Does this test work? What are we testing by changing this?

Reply:
Yes, the test is working. it was proved that the field "username" is mandatory, because this field was sent and only this field should be verified.

the previous test verified that the other required fields were missing.